pom.xml

```diff
@@ -111,6 +110,7 @@
     <play2-json.version>2.6.13</play2-json.version>
     <play2-ebean.version>4.1.3</play2-ebean.version>
     <play2-enhancer.version>1.2.2</play2-enhancer.version>
+    <play2-jdbc-evolutions.version>2.6.13</play2-jdbc-evolutions.version>
```

Comment:
It was required based on code analysis; I'm guessing because it's referenced as the `DynamicEvolutions` injection dependency as per Play documentation. As discussed, I don't believe it's correct the way it is, but it does seem to work.